Makefile

```diff
@@ -4,20 +4,20 @@ VERSION = 1.2.1
 GOMODULE = github.com/kubeopsskills/cloud-secret-resolvers/cmd/csr
 
 buildWindows:
-	env GOOS=windows $(GOARCH) go build -o ./$(TARGET_PATH)/windows/csr.exe $(GOMODULE)
-	cd $(TARGET_PATH) && zip csr-Windows-$(VERSION).zip ./windows/csr.exe
+	env GOOS=windows $(GOARCH) go build -o ./$(TARGET_PATH)/csr.exe $(GOMODULE)
+	cd $(TARGET_PATH) && zip csr-Windows-$(VERSION).zip ./csr.exe
 
 buildMacOS:
-	env GOOS=darwin $(GOARCH) go build  -o ./$(TARGET_PATH)/macos/csr $(GOMODULE)
-	cd $(TARGET_PATH) && tar -zcvf csr-MacOS-$(VERSION).tar.gz ./macos/csr
+	env GOOS=darwin $(GOARCH) go build  -o ./$(TARGET_PATH)/csr $(GOMODULE)
```

Comment:
For MacOS, Linux, Arm, in the build process, CSR will be built parallel for those platforms and the binary will be conflict.